vagrant_support/checksetup_answers.j2

```diff
@@ -0,0 +1,37 @@
+$answer{'ADMIN_EMAIL'}          = 'vagrant@{{ WEB_HOSTNAME }}';
+$answer{'ADMIN_OK'}             = 'Y';
+$answer{'ADMIN_PASSWORD'}       = 'vagrant01!';
```

Comment:
If there is any possible way we can generate this and write it to a file on disk and tell the user to read that file for it, and/or read that file ourselves in the vagrant scripts, that would be a lot fewer shared-password-vulnerable BMO vagrant installs.